.github/stale.yml

```diff
@@ -0,0 +1,26 @@
+daysUntilStale: 300
+daysUntilClose: 30
+only: pulls
+limitPerRun: 1
+staleLabel: stale
+
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  Hi friend! We really, really, really appreciate that you have taken the time
+  to make a PR/issue on `conda-forge/staged-recipes`! `conda-forge` only exists
+  because people like you donate their time to build and maintain conda recipes
+  for use by the community. In an effort to maintain this repository and increase
+  the signal-to-noise for open PRs/issues, the maintainers of `staged-recipes` close
+  excessively old PRs/issues after six months. This PR/issue will remain open
```

Comment:
Should this mention that these PRs likely need rebuilding to make sure they work?

Also is it worth mentioning the bot command to ping the team?

Reply:
Yeah that sounds great! Do you want to take a pass at adding this info?

Reply:
Alright, what do you think of the new text?

Reply:
Looks great, thanks 👍